static/src/javascripts/bootstraps/video-embed.js

```diff
@@ -9,7 +9,6 @@ define([
     'common/utils/config',
     'common/utils/defer-to-analytics',
     'common/utils/template',
-    'common/modules/analytics/omniture',
```

Comment:
Omniture was removed from `video-embed` last week - https://github.com/guardian/frontend/pull/14731/files. You'll probably want to rebase 😄